typings/index.d.ts

```diff
@@ -3294,6 +3302,8 @@ declare module 'discord.js' {
     type?: OverwriteType;
   }
 
+  type NSFWLevel = keyof typeof NSFWLevels;
```

Comment:
This should be moved above `OverwriteData`

Reply:
👍